userspace/sysdig/csysdig.cpp

```diff
@@ -700,17 +700,6 @@ sysdig_init_res csysdig_init(int argc, char **argv)
 				bpf = true;
 				bpf_probe = probe;
 			}
-			else
-			{
-				const char *home = std::getenv("HOME");
```

Comment:
We don't need to load default bpf probe path when not in bpf mode (ie: when `SYSDIG_BPF_PROBE` is not set).
We need to load it:
* when SYSDIG_BPF_PROBE is empty
* when `--bpf` optional argument is empty (ie: `--bpf ""`)
Basically, whenever we are in bpf mode AND bpf_probe is an empty string.